framework/include/materials/Material.h

```diff
@@ -342,7 +396,7 @@ template<typename T>
 MaterialProperty<T> &
 Material::declarePropertyOld(const std::string & prop_name)
 {
-  registerPropName(prop_name, false, Material::OLD);
+  registerSuppliedPropName(prop_name, Material::OLD);
   return _material_data.declarePropertyOld<T>(prop_name);
 }
```

Comment:
Looks like `declarePropertyOld` and `declarePropertyOlder` no longer add the property to `_supplied_props`. Maybe this is intentional but the old code path appears to do that, but not the new one.